src/screens/Home/Home.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { Box, Image, Text } from 'grommet'
+import Kelp from 'images/kelp.png'
+import Title from 'images/title.svg'
+import ResponsiveImage from 'components/ResponsiveImage'
+import styled from 'styled-components'
+import ChooseLocation from './components/ChooseLocation'
+import { ZooniverseLogotype } from '@zooniverse/react-components'
+
+const StyledText = styled(Text)`
+  font-family: Neuton;
+`
+
+export default function Home() {
+  return (
+    <Box background='sand' direction='row'>
+      <Box
+        basis='1/4'
+        border={{ color: 'kelp', side: 'right', size: 'large' }}
+      >
+        <Image a11yTitle='Kelp Background' fit='cover' src={Kelp} />
```

Comment:
Images used for design rather than content don't need alt tags. What you can do to let the screen reader know not to read this out is to use `alt=''`.